src/main/java/com/zendesk/maxwell/schema/SchemaPosition.java

```diff
@@ -85,7 +87,7 @@ private void store(BinlogPosition newPosition) {
 		if ( newPosition == null )
 			return;
 
-		String sql = "INSERT INTO `maxwell`.`positions` set "
+		String sql = "INSERT INTO `" + this.schemaDatabaseName + "`.`positions` set "
```

Comment:
all of these look like potential SQL injection sites to me.  Maybe we instead of substituting in the dbName we strip off the explicit db name and connect the `ConnectionPool` to a particular db?  (if that's possible...)